python/py-sure/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-sure
-version             1.4.11
+version             2.0.0
 revision            0
 
 platforms           darwin
```

Comment:
the `platforms` field is set as default to `darwin` since MacPorts 2.7 so you can remove this line